tools/background_removal/background_removal.xml

```diff
@@ -0,0 +1,55 @@
+<tool id="background_removal" name="Background Removal" version="0.1">
```

Comment:
```suggestion
<tool id="background_removal" name="Background Removal" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="20.05">
```
The versioning scheme is suggested by the IUC: https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#tool-versions

Reply:
Done